.github/workflows/build_and_test.yml

```diff
@@ -172,26 +219,81 @@ jobs:
         include:
           # AArch64
           - arch: aarch64
+            compiler: gcc
             qemu_cpu: "max,sve=off"
           - arch: aarch64
+            compiler: gcc
             qemu_cpu: "max,sve=on,sve128=on"
           - arch: aarch64
+            compiler: gcc
             qemu_cpu: "max,sve=on,sve256=on"
           - arch: aarch64
+            compiler: gcc
             qemu_cpu: "max,sve=on,sve512=on"
+          # Some tests fail when compiled with LLVM only
+          # - arch: aarch64
+          #   compiler: llvm
+          #   qemu_cpu: "max,sve=off"
+          # - arch: aarch64
+          #   compiler: llvm
+          #   qemu_cpu: "max,sve=on,sve128=on"
+          # - arch: aarch64
+          #   compiler: llvm
+          #   qemu_cpu: "max,sve=on,sve256=on"
+          # - arch: aarch64
+          #   compiler: llvm
+          #   qemu_cpu: "max,sve=on,sve512=on"
           # Aarch32
           - arch: armhf
+            compiler: gcc
+            binfmt: arm
```

Comment:
Would you be able to use the same layout as in `build-cross` ? As in have:
```
      matrix:
        arch: [aarch64, armhf, ppc64el, s390x, riscv64]
        compiler: [gcc, llvm]
```
and specifying `qemu_cpu` and `binfmt` in a similar way as `gnupkg`, while excluding some llvm and gcc variants?

Reply:
Defining multiple `include` with only the `qemu_cpu` being different will not trigger different builds unfortunately. The easiest way to trigger all the builds we want is to do is this way.

Reply:
I'm trying something out at https://github.com/rivosinc/sleef/compare/upstream-riscv...upstream-riscv-2, I'll merge into this PR if it's working.

Reply:
Sure! Let me know how this goes!

Reply:
It doesn't work: https://github.com/rivosinc/sleef/actions/runs/6867005626/job/18676581199